src/components/UI/NetworkBar/NetworkBar.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import styled from "@emotion/styled";
+import { vars } from "../../../styles";
+import { SvgIcon, SvgIconX } from "../SvgIcon";
+
+const NetworkWrap = styled.div`
+  background-color: #001f29;
```

Comment:
this bg can put into src/tailwind.js, then also import into vars then next time can use liao.